### DISCUSSION TITLE
Printing text

### DISCUSSION THREAD

**Discussion by claudiorrrr** [NaitLee/Cat-Printer#19] [Q&A] [answered]
Is it any way to print text?

(I finally got the printer to work!)

**Comment by NaitLee** [comment]
I remember you have Mac and like command line :)

The command line now have -t switch to print text, ...

EDIT: ellipsis...

**Reply by NaitLee** [reply]
For Web UI there dont have one yet. maybe it's time to start.
There are many tricks to turn text to image, though

**Comment by NaitLee** [comment, marked as answer]
Well there's #11. See that.
